apis/fluentbit/v1alpha2/plugins/output/prometheus_remote_write_types.go

```diff
@@ -0,0 +1,93 @@
+package output
+
+import (
+	"fmt"
+
+	"github.com/fluent/fluent-operator/apis/fluentbit/v1alpha2/plugins"
+	"github.com/fluent/fluent-operator/apis/fluentbit/v1alpha2/plugins/params"
+	"github.com/fluent/fluent-operator/pkg/utils"
+)
+
+// +kubebuilder:object:generate:=true
+
+// An output plugin to submit Prometheus Metrics using the remote write protocol
+type PrometheusRemoteWrite struct {
```

Comment:
Why does it not support `workers`?

<img width="583" alt="1661216556(1)" src="https://user-images.githubusercontent.com/53003665/186045598-9c8f62a1-dfc3-4396-bd45-c529ea6dfeee.png">